### PR TITLE
influxdb 0.9.3-rc1

### PR DESCRIPTION
**Pull request by gunnaraasen**
Also includes an update to InfluxDB dependencies.

### CODE REVIEW

File: Library/Formula/influxdb.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Influxdb < Formula
 
   stable do
     url "https://github.com/influxdb/influxdb/archive/v0.9.2.tar.gz"
-    sha256 "5e0f33c3d4076509118ad9b02c61f64f7db138ada141675c84a5326e77b21115"
+    sha256 "8fc6f9df9af4a97fa38ae548e3a236e8ae1c79c5655d939c04d87e0d04be11e8"
```
Comment (bfontaine) [inline Library/Formula/influxdb.rb]: Why did this change?

Comment (bfontaine) [inline Library/Formula/influxdb.rb]: It seems to be an issue on your side.

```
$ curl -sL https://github.com/influxdb/influxdb/archive/v0.9.2.tar.gz | shasum -a 256
5e0f33c3d4076509118ad9b02c61f64f7db138ada141675c84a5326e77b21115  -
```

Comment (gunnaraasen) [inline Library/Formula/influxdb.rb]: That was an accident. It's been switched back.

**Comment by gunnaraasen** [comment]
@bfontaine Fixed the error. Does this look good now?

**Comment by mikemcquaid** [comment]
Just need to trigger a rebuild and will merge this. Thanks!

**Comment by DomT4** [comment]
I triggered a rebuild here ~10 minutes ago, per the same issue I presume you encountered. Apologies for not posting re that.

**Comment by mikemcquaid** [comment]
@DomT4 np!

**Comment by DomT4** [comment]
:rocket: in https://github.com/Homebrew/homebrew/commit/bf6d702ed2f5023205e4ea351e7fa3d0c5ff47b9. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Comment by DomT4** [comment]
Oh, future nit: Please tag devel updates with `(devel)` at the end of the commit message, Thanks!

**Comment by gunnaraasen** [comment]
Thanks! I'll add `(devel)` on the next rc update.

**Pull request closed without merging**